tests/virt_autotest/install_package.pm

```diff
@@ -107,17 +107,14 @@ sub install_package {
 }
 
 sub run {
-    if (is_x86_64) {
-        script_run("zypper ar -p 80 http://download.suse.de/ibs/home:/Julie_CAO:/branches:/workaround/SLE-15-SP6 workaround");
-        script_run("zypper --gpg-auto-import-keys ref -r workaround");
-        save_screenshot;
-    }
+    script_run("zypper ar -p 80 http://download.suse.de/ibs/home:/Julie_CAO/SLE-15-SP6 julie");
```

Comment:
Quick question, why do we use with - http://download.suse.de/ibs/home:/Julie_CAO/SLE-15-SP6 here? Is it the workaround or other?  

http://download.suse.de/ibs/home:/Julie_CAO should be the personal repo. We'd better to continue to use with - http://dist.nue.suse.com/ibs/QA:/Head/

Meanwhile, this repo should be work with SLE15SP6 version - http://download.suse.de/ibs/home:/Julie_CAO/SLE-15-SP6

Please add some info or comments here. Thanks

Reply:
It is the workaround to run tests with https://github.com/SUSE/qa-automation/pull/824. It will be removed after validation runs finish. No worries about it, we will still use http://dist.nue.suse.com/ibs/QA:/Head/

Reply:
Will merge firstly https://github.com/SUSE/qa-automation/pull/824.. Suggest to run at least one more key run to see if using QA head is as expected.

Reply:
I'll have another MR about adding that new settings(two keys) in our job group yaml file, that MR should be merged firstly, then these two PRs.

Reply:
Noticed, will pay attention to merge order